### DISCUSSION TITLE
Block Stale Images

### DISCUSSION THREAD

**Discussion by challakiran334** [kyverno/policies#987] [Q&A]
In kyverno policy reporter UI, in block stale images (rule is set for 2 months for older images) but report shows the latest images which were pushed recently.

**Comment by chipzoller** [comment]
Converted to discussion. I assume you're referring to [this policy](https://kyverno.io/policies/other/block-stale-images/block-stale-images/). If so, it looks at the created date for the image and not the push date.

**Reply by chipzoller** [reply]
Is this question resolved?
